waspc/packages/deploy/src/providers/fly/GlobalOptions.ts

```diff
@@ -0,0 +1,5 @@
+export interface GlobalOptions {
+  waspExe: string
```

Comment:
I thought I replied to this one, but don't see the comment here. 🤔  Main reason for exe instead of bin to me is all exes are binary, but not all binaries are executable (like .dll and .so files). I think exe is prob safer here.